### DISCUSSION TITLE
Update to version 7.4.7 causing error

### DISCUSSION THREAD

**Discussion by richardcgarmstrong** [MasaCMS/MasaCMS#344] [Q&A]
Hello, 
I have updated a 7.4.4 server to 7.4.7 which appears to have completed successfully.  However, I am getting strange errors as the CMS is no longer properly handling m-tags.  All m-tags are not being rendered, but are rather, the actual [m] tag markup is appearing on the web page ([m]$.dspInclude('modules/custom/CONTENT_Agenda-SWAC.cfm')[/m]).

Unsure how to correct this issue and I am force to roll-back to a version which requires a critical security update.  I would be thankful for any help.

**Comment by jochemd** [comment]
The m-tag has been [disabled by default in 7.4.6](https://docs.masacms.com/release-notes/#whats-changed-1).

Try enabling them [in the settings file](https://github.com/MasaCMS/MasaCMS/commit/fb27f822fe426496af71205fa35208e58823fcf6#diff-e9e6c08919948c26bc7170651166df3ae48a3ec28e1184d64ce4900db933bcbb).

**Reply by mctunes** [reply]
Hi,

We are also getting the [m] tag markup rendered on pages, rather than the cfm file actually getting included. `enablemuratag=true` is set correctly in `config/settings.ini.cfm`.

I should add that we are in the process of moving our content from Mura 7.1 to Masa 7.4.8, and the `dspInclude` code we were using was:

`[mura]dspInclude('themes/test/display_objects/external/clients/dsp_client.cfm')[/mura]`

I updated the code as per the documentation, but this also fails to render correctly:

`[m]$.dspInclude('themes/test/display_objects/external/clients/dsp_client.cfm')[/m]`

Am I missing something, or is it possible a bug might have been introduced in 7.4.7?

Best regards,
Mike.

**Reply by mctunes** [reply]
OK, after digging through the ContentRenderer code, it looks like `enableDynamicContent=true` also needs to be set in `config/settings.ini.cfm` in order to get the [m] tags to be parsed correctly. After adding this and reloading the application, the content was displayed correctly.

Is this by design, or should having only `enablemuratag=true` be sufficient?
